### ISSUE TITLE
Empty form does not show a box for "put elements in here"

### ISSUE THREAD

**Issue by KuenzelIT** [kevinchappell/formBuilder#610]
### Description:
If a form is empty, there should be this "stage" box with a min-height of at least 100px or something. Instead, the element itself has a min-height of 0px (not by CSS class, but the element itself). So it's not a box but a line with text on it instead. Looks strange and doesn't seem to be on purpose.

EDIT: it works on readthedocs correctly.

### Environment Details:

 - formBuilder Version: 2.9.6
 - Browser: Firefox & Chrome
 - OS: Windows 7

### Expected Behavior
See a box

### Actual Behavior
Only a dashed line with text on it.

### Steps to Reproduce
Simply use the form builder without any elements on the form.

### Screenshot - *(optional)*
[https://imgur.com/kFUCF0X](https://imgur.com/kFUCF0X)

**Comment by BrianHVB** [comment]
I have the same issue. There are two workarounds, but neither is ideal:

<i>#</i>1
---
Remove or comment out line 1369 in form-builder.js
https://github.com/kevinchappell/formBuilder/blob/36d4d993252e172d6d77c562564574ed0daec96d/src/js/form-builder.js#L1361-L1375
As far as I can tell, there are no ill-effects of doing this, and I don't understand why the min-height would be dynamically set from within the plugin.

<i>#</i>2 
---
Override the minimum height by adding the following CSS
```scss
#fb-editor {
        // customize look and feel

	.frmb.ui-sortable {
		min-height: 150px !important;
	}
}
```

I really don't like either solution. Commenting out lines of code may introduce bugs down the line. Even when it seems the added code doesn't do much, it was likely put in place to fix some other one-off bug.

But I really, really don't like using `!important` in CSS. It's a huge code-smell and almost always indicates that something is either badly designed or that someone is being lazy.

**Comment by kevinchappell** [comment]
Does this happen when the form is emptied or does it need to be loaded with specific data? I'm having trouble reproducing the issue.

**Comment by BrianHVB** [comment]
This happens for me before any data has been added to the form. My test form looks like this:

```javascript
class TestBuilder {
	el = null;
	constructor() {
		$(this.initializeBuilder);
	}

	initializeBuilder = (e) => {
		$('#fb-editor').formBuilder();
	};

	showEditorClicked = (e) => {
		$('#fb-editor').toggle();
	};

	render() {
		return (
			<div ref={el => this.el = $(el)}>
				<button id="fb-toggle-editor" onClick={this.showEditorClicked}>toggle editor</button>
				<div id="fb-editor">editor</div>;
			</div>
		)
	}
}

const target = $('#template-content-container');
const tb = new TestBuilder();
ReactDOM.render(tb.render(), target[0]);
```

I like writing my HTML inside my .js files using JSX so I'm using ReactDOM to render JSX to a container (but without using any React state or other Reacty things). By the time any interaction with jQuery and/or FormBuilder occurs, the DOM has already been updated, so I don't think this is causing the issue.

**Comment by kevinchappell** [comment]
Can you initialize formBuilder in `componentDidMount`? I never used jQuery in react app before and having a hard time setting up an example but maybe we could collab on this? https://codesandbox.io/s/k058xq493

**Comment by kevinchappell** [comment]
I agree, there should be a React example for formBuilder as well.

Regarding the problem, `$stage.css('min-height', $cbUL.height()) ` is for sure causing the problem. When it reads `$cbUL.height()` it's `0` until the plugin finishes loading. It's "fixed" in release/2.10.0 branch by wrapping it in a `setTimeout` (:expressionless:) but I'd like to remove it completely and just make the wrapper `display:flex;`. The problem is that could be a breaking change for older browsers.

**Comment by BrianHVB** [comment]
I modified your React/FormBuilder example on CodeSandbox. Took a bit of fiddling, but I have it running. 
https://codesandbox.io/s/6zl7o2jynn

A couple of thoughts.

When using jQuery libraries with ES6 projects (especially React), you often need to explicitly set the global variable `$`. For good measure, I also set `jQuery`. ES6 imports don't modify the global namespace, so if you do an `import abc from 'xxx'` then later load another library that expects `abc` to be defined as a certain thing, that library probably won't work.

To avoid this issue, jQuery libraries should use the [non-conflict style](https://api.jquery.com/jQuery.noConflict/) for extentions in ES6, and always wrap the use of `$`, e.g.

```
import jQuery from 'jquery';
jQuery(function($) {
     // all code that needs to use $ goes inside
});
```
As for the example, the ordering of the imports/requires is very sensitive. I haven't look much at the code for either `formBuilder` or `jquery-ui`, but it seems that both rely on the global `$` and so they must be required **after** it is set. The import ordering also seems dependent on the build envrionment. For instance, the CodeSandbox IDE example will only work if I `require('jquery-ui-sortable)` but in my Babel/Webpack project, I can `import 'jquery-ui-sortable'` instead.

I also changed how `refs` were used. React essentially has two ways of assigning reference elements. The depreciated format would assign directlyand could make use of `this.refs`. The current style uses functions instead.
